music21/braille/segment.py

```diff
@@ -1670,12 +1670,8 @@ def prepareSlurredNotes(music21Part, **keywords):
     for slur in music21Part.spannerBundle.getByClass(spanner.Slur):
         firstNote = slur[0]
         lastNote = slur[1]
-
-        try:
-            beginIndex = allNotes.index(firstNote)
-            endIndex = allNotes.index(lastNote)
-        except exceptions21.StreamException:
-            continue
```

Comment:
what if the note that a Spanner refers to has been removed from the Stream?  Like:

```
>>> n = note.Note('C')
>>> sp = spanner.Slur([n])
>>> s = stream.Stream()
>>> s.append([sp, n])
>>> s.remove(n)
>>> { call the braille segment routine }
```

Reply:
Yep, I'll revert. A warning might be nice.

Reply:
But in that case indexing into the slur will fail, and that's not wrapped. We should probably be using `.getFirst()` and wrapping in a try.